src/explore-education-statistics-admin/src/components/PageHeader.tsx

```diff
@@ -117,6 +117,13 @@ interface LoggedInLinksProps {
 
 const LoggedInLinks = ({ user }: LoggedInLinksProps) => (
   <>
+    {user.permissions.canAccessSystem && (
```

Comment:
Is this is the right permission? 

We have pre-release only users who should only have limited access to see specific releases. These types of users typically live outside of the statistics publishing team.

I'd imagine we probably shouldn't be letting these users see this type of information.

The `canAccessAnalystPages` permission is probably more appropriate.